function addTwoNums(a,b){
    try{
    console.log(a+b);
    } 
    catch(error){
        console.log("error:", error);
    }
}
addTwoNums(5,"5")
